### DISCUSSION TITLE
It's compatible with W25Q64JW?

### DISCUSSION THREAD

**Discussion by rportino** [nofeletru/UsbAsp-flash#81] [Q&A]
It's Asprogrammer compatible with W25Q64JW?

**Comment by nofeletru** [comment]
Yes.
But note that the voltage of this memory is 1.8 volts.
For example, the CH341A outputs 3.3 volts by default. So you will need an adapter from 3.3/5 volts to 1.8 volts.

**Comment by nofeletru** [comment]
I have updated chiplist.xml

**Comment by rportino** [comment]
Thank you Alexander.
    On Friday, March 17, 2023 at 12:43:51 PM GMT-3, Alexander ***@***.***> wrote:  
 
 


I have updated chiplist.xml
